### DISCUSSION TITLE
is it possible to somehow to play or demux unfinished file?

### DISCUSSION THREAD

**Discussion by galactic76** [justdan96/tsMuxer#775] [Q&A]
Hi!
Since rarbg is died, i have several 99.9 remuxes that freezes like this 

is it possible to somehow to play or demux unfinished mkv or mp4 file?

Thanks

**Comment by galactic76** [comment]
now it stops with 

MP4/MOV error: Invalid H.264/AVC frame at position 1169922797

**Comment by trzyglow** [comment]
unfinished files are corrupt files, so even if you manage to demux them (with tsmuxer, mkvtoolnix etc.) the extracted files will still be broken in some regard. (if you're lucky it's just an obscure encoder error and re-encoding with ffmpeg fixes the issue)
